gpt4all-chat/src/xlsxtomd.cpp

```diff
@@ -25,7 +25,7 @@ static QString formatCellText(const QXlsx::Cell *cell)
 {
     if (!cell) return QString();
 
-    QVariant value = cell->value();
+    QVariant value = cell->readValue();
```

Comment:
readValue also returns the original text of the formula instead of the computed result. This might be useful in some cases, but seems like the wrong default for a table full of computed numbers.

Reply:
What if we used `cell->dateTime().toDateTime()` instead of `value.toDateTime();`?